controllers/numaresourcesscheduler_controller.go

```diff
@@ -194,3 +200,24 @@ func (r *NUMAResourcesSchedulerReconciler) updateStatus(ctx context.Context, sch
 	}
 	return nil
 }
+
+// a reference which illustrates such technique can be found at https://blog.questionable.services/article/kubernetes-deployments-configmap-change/
+func (r *NUMAResourcesSchedulerReconciler) syncConfigMapHash(ctx context.Context) (string, error) {
+	mfConfigMap := r.SchedulerManifests.ConfigMap // shortcut
+	updatedConfigMap := &corev1.ConfigMap{}
+	key := client.ObjectKey{
+		Namespace: mfConfigMap.Namespace,
+		Name:      mfConfigMap.Name,
+	}
+	if err := r.Get(ctx, key, updatedConfigMap); err != nil {
```

Comment:
minor: I'd have used `client.ObjectKeyFromObject` but not worth a resubmit for this

Reply:
blessing change, resubmiting anyway